packages/components/src/components/hds/layout/grid/index.ts

```diff
@@ -82,15 +83,36 @@ export default class HdsLayoutGrid extends Component<HdsLayoutGridSignature> {
     }
   }
 
+  /*
+    LOGIC:
+
+    If columnMinWidth is passed in:
+    1) we set --hds-layout-grid-column-min-width to the passed in value
+    2) We use the fallback value of "auto-fit" for --hds-layout-grid-column-fill-type (reults in a more fluid layout)
+
+    If columnWidth is passed in:
+    1) we set --hds-layout-grid-column-min-width to the passed in value
+    2) we add a class name to the Grid which sets --hds-layout-grid-column-fill-type to "auto-fill" (results in a more fixed layout)
```

Comment:
[question] instead of setting a class that sets a variable that sets a value, why not directly set the `--hds-layout-grid-column-fill-type` to `auto-fill` directly here (same as what is done for the `--hds-layout-grid-column-min-width`)? in any case, `--hds-layout-grid-column-fill-type` has a fallback value in CSS

Reply:
(it would also take away a bit of indirection, that at first took me a while to understand)